pkgs/tools/security/pass/default.nix

```diff
@@ -31,7 +31,8 @@ let
       selected = [ pass ] ++ extensions passExtensions
         ++ lib.optional tombPluginSupport passExtensions.tomb;
     in buildEnv {
-      name = "pass-extensions-env";
+      # lib.getExe looks for name, so we keep it the same as mainProgram
+      name = "pass";
```

Comment:
but we are setting it explicitly below.

Reply:
No, they are two different derivations. This one is in `passthru` of the `pass` derivation.

Reply:
If this change is removed, then:
```console
nix-repl> lib.getExe (pass.withExtensions ( exts: [ exts.pass-import ])) 
"/nix/store/wxv2z5mi10z6khjd2x79lkgyhc68zi05-pass-extensions-env/bin/pass-extensions-env"
```